README.md

```diff
@@ -7,10 +7,98 @@ new [cisagov](https://github.com/cisagov) Go project started.
 This skeleton project contains [licensing information](LICENSE), as
 well as [pre-commit hooks](https://pre-commit.com) and
 [GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
+appropriate for a Go package.
 
-In many cases you will instead want to use one of the more specific
-skeleton projects derived from this one.
+## Installation ##
+
+To install the example package, use `go build`:
+
+```console
+cd src/example
+go build
+```
+
+This will produce an executable `example` within the current directory.
+
+Execute the `./example` binary to run the program.
+
+You should see the contents of the main package as defined in [main.go](src/example/main.go).
```

Comment:
I do not believe this is correct information based on the current state of the repository.

Reply:
Please see https://github.com/cisagov/skeleton-golang-package/pull/2/commits/7a02cf40868c3c4479826ebdc47cb242e1d10c45

Reply:
Thank you for updating the binary name but there is no `src/example` directory in this branch. As far as I can tell you would run `go build` in the repository root.

Reply:
Right, that's old information based on the initial structure. Please see: https://github.com/cisagov/skeleton-golang-package/commit/f03e6441ec610e3fb09aca074001ca326c1f942e